### PR TITLE
[fix] Optimize instructions based on feedback for contracts

### PR DESCRIPTION
**Pull request by Sneh1999**
**What changed? Why?**
- Removes all warnings on build
- Adds clearer instructions on deploying your contract, creating an abi etc
- Adds a console.log to let the user know that contracts are loading when the user runs `npx @coinbase/build-onchain-apps@latest create`
- Formatted all contracts

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: apps/build-onchain-apps/contracts/src/CustomERC1155.sol [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
-    ERC1155(
-      'https://ipfs.io/ipfs/bafybeiexud2pveyr2ybelpjvwzhpyhw57tefraibf7jn4cecol6qu3jiau/1.json'
-    )
-  {}
+    constructor() ERC1155("https://ipfs.io/ipfs/bafybeiexud2pveyr2ybelpjvwzhpyhw57tefraibf7jn4cecol6qu3jiau/1.json") {}
 
-  function mint(address account, uint256 id, uint256 amount, bytes memory data) public {
-    _mint(account, id, amount, data);
-  }
+    function mint(address account, uint256 id, uint256 amount, bytes memory data) public {
```
Comment (Zizzamia) [inline apps/build-onchain-apps/contracts/src/CustomERC1155.sol:9]: Is it a thing that Solidity use 4 spaces instead of 2? cc @Sneh1999

Comment (Sneh1999) [reply apps/build-onchain-apps/contracts/src/CustomERC1155.sol:9]: yes, seems like it: https://docs.soliditylang.org/en/latest/style-guide.html#indentation

Comment (Zizzamia) [reply apps/build-onchain-apps/contracts/src/CustomERC1155.sol:9]: that's such a Java vibes for 4 spaces. Not sure how to feel about that. eheheh

**Review by Zizzamia — APPROVED** [review]
Niiice!

**Pull request merged**